content/administration/maintain/on_premise.rst

```diff
@@ -143,25 +132,24 @@ Duplicate a database
 ====================
 
 You can duplicate your database by accessing the database manager on your
-server (<odoo-server>/web/database/manager). In this page, you can easily
+server (<odoo-server>/web/database/manager). On this page, you can easily
 duplicate your database (among other things).
 
 .. image:: on_premise/db_manager.gif
    :align: center
 
-When you duplicate a local database, it is **strongly** advised to change
-the duplicated database's uuid (Unniversally Unique Identifier), since this
-uuid is how your database identifies itself with our servers. Having two
-databases with the same uuid could result in invoicing problems or registration
-problems down the line.
+When you duplicate a local database, it is **strongly** advised to change the duplicated
+database’s UUID (Unniversally Unique Identifier), since this UUID is how your database identifies
+itself with our servers. Having two databases with the same UUID could result in invoicing problems
+or registration problems down the line.
 
 .. note::
-   From July 2016 onward, Odoo 9 now automatically change the uuid of a duplicated database; a
-   manual operation is no longer required.
+   From July 2016 onward, Odoo 9  and up now automatically changes the UUID of a duplicated database;
+   a manual operation is no longer required.
 
-The database uuid is currently accessible from the menu :menuselection:`Settings --> Technical -->
-System Parameters`, we advise you to use a `uuid generator <https://www.uuidtools.com>`_ or to
-use the unix command ``uuidgen`` to generate a new uuid. You can then simply replace it like any
+The database UUID is currently accessible from the menu :menuselection:`Settings --> Technical -->
+System Parameters`, we advise you to use a `UUID generator <https://www.uuidtools.com>`_ or to
+use the Unix command ``uuidgen`` to generate a new UUID. You can then simply replace it like any
 other record by clicking on it and using the edit button.
 
 .. image:: on_premise/db_uuid.png
```

Comment:
I'm not sure this section is worth keeping, but FYI, the screenshot is way too wide, making it difficult to view on most screens. A crop like this would be sufficient:
<img width="537" alt="firefox_ziRWTLTZbO" src="https://github.com/odoo/documentation/assets/70884563/725839d4-b669-4a06-8e96-932149ac9673">